sources/north-america/us/pa/PEMA_Orthoimagery_2022.geojson

```diff
@@ -3,14 +3,14 @@
     "properties": {
         "id": "PEMA_Orthoimagery_2022",
         "attribution": {
-            "url": "https://www.pema.pa.gov/911-Program/Partners/NG911-GIS/Aerial-Imagery/Pages/default.aspx",
-            "text": "Pennsylvania Emergency Management Agency",
+            "url": "https://www.pasda.psu.edu/uci/DataSummary.aspx?dataset=5177",
+            "text": "PASDA",
```

Comment:
PEMA is the author and PASDA is the host, right? The previous text would be more accurate, although it doesn’t matter that much since attribution isn’t required.